omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -56,16 +56,29 @@ export default class ObsAcrossEncountersController {
   }
 
   fetchEncounters() {
-    this.openmrsRest.get("encounter", {
-      patient: this.config.patientUuid,
-      encounterType: this.config.encounterType ? this.config.encounterType : null,
-      v: 'custom:(uuid,encounterDatetime,obs:(id,uuid,value,concept:(id,uuid,name:(display),datatype:(uuid)),groupMembers:(id,uuid,display,value,concept:(id,uuid,name:(display),datatype:(uuid))))',
-      limit: this.getMaxRecords(),
-      fromdate: this.widgetsCommons.maxAgeToDate(this.config.maxAge),
-      order: this.order
-    }).then((response) => {
-      this.parseEncounters(response.results);
-    });
+    let promises = [];
+    let encounterTypes = this.config.encounterType ? this.config.encounterType.replace(' ', '').split(',') : [null];
```

Comment:
Might want to create a new config paramter "encounterTypes" to make it clear that multiple encounter types are supported... we'd need to keep supporting encounterType for the legacy use case as well, of course.

Reply:
Ok yes. Simple inquiry, what would take precedence in the case where both parameters are provided?

Reply:
EncounterTypes, I'd think, but using both would be an "error" use case so I don't have a strong preference.

Reply:
Ok, thanks @mogoodrich.

Reply:
@mogoodrich when using both, do you prefer an "error" or adding/merging the single one to the list specified through `encounterTypes`?

Reply:
Slight preference for "error", but definitely just a slight preference.

Reply:
@mogoodrich sorry, I have the opposite preference ;-)

If people were to push the config possibilities to their limits, the logical outcome would be to merge them. We will have a similar comment on RA-1738 [here](https://issues.openmrs.org/browse/RA-1738?focusedCommentId=263518&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-263518).

I have the opposite preference, but first of all my preference is that we are consistent. So either we always attempt to merge where possible, or we always threat this as errors.

Reply:
Fine with the merging functionality @mks-d , my preference for an error was definitely very slight... :)

Reply:
Great, that one is for you @Ruhanga. Also let's updated RA-1738's behaviour to be consistent with the choice made here.

Reply:
Cool, thanks @mogoodrich, @mks-d.